data/templates/set_bmc_credentials.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo " Getting the user ID"
+cmdReturn=$(sudo ipmitool user summary)
+myarray=(${cmdReturn//$'\n'/ })
+userNumber=${myarray[8]}
+userNumber=$((userNumber + 1))
+echo "number of user : $userNumber"
```

Comment:
@benbp it will have two entries with the same username.
We can edit the script to overwrite the existing username entry if you want